contracts/contracts/token/OUSD.sol

```diff
@@ -85,8 +95,12 @@ contract OUSD is Initializable, InitializableToken, Governable {
      * @return A uint256 representing the _amount of base units owned by the
```

Comment:
super nit: update the comment. both balanceOf and creditsBalanceOf method say "representing the _amount of base units"